@xen-orchestra/web-core/lib/components/info/UiInfo.vue

```diff
@@ -0,0 +1,56 @@
+<!-- v2 -->
+<template>
+  <div class="vts-ui-info">
+    <UiIcon class="icon" :color :icon />
+    <p class="message"><slot /></p>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import type { IconColor } from '@core/types/color.type'
+import {
+  faCheckCircle,
+  faExclamationCircle,
+  faInfoCircle,
+  faXmarkCircle,
+  type IconDefinition,
+} from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+type Props = {
+  status: 'info' | 'success' | 'warning' | 'danger'
+}
+
+const props = defineProps<Props>()
+
+defineSlots<{
+  default(): any
+}>()
+
+const states: Record<Props['status'], { icon: IconDefinition; color: IconColor }> = {
+  info: { icon: faInfoCircle, color: 'normal' },
+  success: { icon: faCheckCircle, color: 'success' },
+  warning: { icon: faExclamationCircle, color: 'warning' },
+  danger: { icon: faXmarkCircle, color: 'danger' },
+}
+
+const icon = computed(() => states[props.status].icon)
+const color = computed(() => states[props.status].color)
```

Comment:
```suggestion
const icon = computed(() => states[props.accent].icon)
const color = computed(() => states[props.accent].color)
```
Since the possible values for `accent` prop and the `color` values of the icon component are the same, maybe we can return only `states[props.status]` and refactor the `states` object? Or even use `props.accent` for the `:color` prop in the template, line 4?